### PR TITLE
Add docs for Supervisor with Azure AD

### PR DESCRIPTION
**Pull request by benjaminapetersen**
- Note that Azure AD is being rebranded to Entra ID.  I added a second `With Entra ID` page and included a redirect to this `With Azure AD` page.  I imagine in the future we could swap this around, but for now I am sure people are more familiar with Azure AD as a brand.

Menu ordering looks like this:

Screenshot 2023-09-05 at 4 41 36 PM

**Release note**:

```release-note

```

### CODE REVIEW

File: site/content/docs/howto/supervisor/configure-supervisor-with-azuread.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+Create an [OIDCIdentityProvider](https://github.com/vmware-tanzu/pinniped/blob/main/generated/{{< latestcodegenversion >}}/README.adoc#oidcidentityprovider) 
+in the same namespace as the Supervisor.
+
+1. In the [Azure portal](portal.azure.com), navigate to _Home_ > _Azure Active Directory_ > _App Registrations_.
+   1. Copy the `Application (client) ID)` for use in your OIDCIdentityProvider CR later.
+1. Select your application, and then click `Add a certificate or secret`.
+   1. Click `New client secret`, provide a name, an expiration time, and click `create`.
+   1. Copy the secret `Value` for use later with your `OIDCIdentityProvider`. 
+1. Select your application, and then click `Endpoints`.
+   1. Under `Endpoints`, find hte `OpenID Connect Metadata Document` URL.
```
Comment (joshuatcasey) [inline site/content/docs/howto/supervisor/configure-supervisor-with-azuread.md:59]: type, `hte` -> `the`

File: site/content/docs/howto/concierge/configure-concierge-supervisor-jwt.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ cascade:
   layout: docs
 menu:
   docs:
-    name: JWT Authentication with Supervisor
+    name: With Supervisor
```
Comment (benjaminapetersen) [inline site/content/docs/howto/concierge/configure-concierge-supervisor-jwt.md:8]: Word smash, so I adjusted given our Supervisor section is heavily utilizing the `with xyz` pattern now.
Index page title is still full text.

Screenshot 2023-09-05 at 4 26 17 PM

File: site/content/docs/howto/supervisor/configure-supervisor-with-azuread.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+1. Optionally, just use the `Default Directory` that is already created.
+1. Users can be added to the directory via the  `Manage` > `Users` link.
+1. Create a new app:
+   1. Click `App Registrations`.
+   1. Click `New Registration`.
+   1. Enter a `user-facing display name`.
+   1. Choose supported account types.
+   1.  Enter the `Redirect URI`. The redirect uri will be the `spec.issuer` you configured in your `FederationDomain` appended with `/callback`.
+      1. This step can optionally be done later, however login will not work until a valid URL is provided.
+      1. There is no `Front Channel Logout URL` for the Pinniped Supervisor, so again use `spec.issuer` and append with `/logout`.
```
Comment (joshuatcasey) [inline site/content/docs/howto/supervisor/configure-supervisor-with-azuread.md:45]: Is this field required? Should it be left blank?

**Review by joshuatcasey — APPROVED** [review]

File: site/content/docs/howto/supervisor/configure-supervisor-with-azuread.md (hunk 1 above)
Comment (benjaminapetersen) [reply site/content/docs/howto/supervisor/configure-supervisor-with-azuread.md:59]: 👍

File: site/content/docs/howto/supervisor/configure-supervisor-with-azuread.md (hunk 3 above)
Comment (benjaminapetersen) [reply site/content/docs/howto/supervisor/configure-supervisor-with-azuread.md:45]: You can't leave it blank, unfortunately.  But we also don't support it. This seemed to be the most reasonable workaround.

File: site/content/docs/howto/supervisor/configure-supervisor-with-azuread.md (hunk 1 above)
Comment (joshuatcasey) [reply site/content/docs/howto/supervisor/configure-supervisor-with-azuread.md:59]: haha there was a typo in my response.... `type` -> `typo`

**Pull request merged**